python/pyspark/tests/test_rdd.py

```diff
@@ -764,7 +764,7 @@ def test_overwritten_global_func(self):
         # Regression test for SPARK-27000
         global global_func
         self.assertEqual(self.sc.parallelize([1]).map(lambda _: global_func()).first(), "Hi")
-        global_func = lambda: "Yeah"
+        global_func = lambda: "Yeah"  # noqa: E731
```

Comment:
This keeps lambda because it is simple for test?

Reply:
Oh yeah. I didn't change just because I was just worried that changing this to `def` would affect what this targets to test (it has to explicitly assign the function to the global variable).